amazon/ionbenchmark/ion_benchmark_cli.py

```diff
@@ -360,55 +646,50 @@ def ion_python_benchmark_cli(arguments):
     # initialize options that might show up multiple times
     api = [*set(arguments['--api'])] if arguments['--api'] else [API.DEFAULT.value]
     format_option = [*set(arguments['--format'])] if arguments['--format'] else [Format.DEFAULT.value]
+    io_type = [*set(arguments['--io-type'])] if arguments['--io-type'] else [Io_type.DEFAULT.value]
     # option_configuration is used for tracking options may show up multiple times.
-    option_configuration = [api, format_option]
+    option_configuration = [api, format_option, io_type]
     option_configuration_combination = list(itertools.product(*option_configuration))
 
-    print(option_configuration_combination)
-    if arguments['read']:
-        # initialize benchmark report table
-        table = identify_report_table('read')
-
-        for each_option in option_configuration_combination:
-            # reset
-            api, format_option = reset_for_each_execution(each_option)
-
+    # initialize benchmark report table
+    table = identify_report_table(command)
+
+    for each_option in option_configuration_combination:
+        print(f'Generating option {each_option}...')
+        # reset each option configuration
+        api, format_option, io_type = reset_for_each_execution(each_option)
+        binary = format_is_binary(format_option)
+        # TODO. currently, we must provide the tool a corresponding file format for read benchmarking. For example,
```

Comment:
The tool *to convert to* a corresponding file format for read benchmarking. ?

Reply:
Fixed.